AutoRest/Generators/AcceptanceTests/server/routes/azureUrl.js

```diff
@@ -0,0 +1,30 @@
+var express = require('express');
+var router = express.Router();
+var util = require('util');
+var constants = require('../util/constants');
+var utils = require('../util/utils')
+
+var azureUrl = function (coverage) {
+  coverage['SubscriptionIdAndApiVersion'] = 0;
+  router.get('/:subscriptionId/resourcegroups/:resourceGroup', function (req, res, next) {
+    var subscriptionId = req.params.subscriptionId;
+    var apiVersion = req.query['api-version'];
+    var groupName = req.params.resourceGroup;
+    var queryParamCount = Object.keys(req.query).length;
+    if (!subscriptionId || typeof subscriptionId !== 'string') {
+      utils.send400(res, next, 'The provided subscriptionId ' + util.inspect(subscriptionId) + 
```

Comment:
we should be specific about the values we are expecting to receive and will send